lib/processes/visTestPattern.cpp

```diff
@@ -463,23 +466,31 @@ void visTestPattern::compute_expected_data() {
     size_t num_freqs = freqs.size();
     size_t num_prods = prods.size();
 
+    INFO("Computing expected data...");
```

Comment:
To me this seems like more of a debug statement (same with the second one).  Maybe switch this to debug, but then set the config to make this stage run at the `debug` log level if this is useful.

Reply:
I left it there as INFO, because when run without OMP, this takes around 10min and drops all frames in that time.